zadanie_8_9/zadanie_8_9.js

```diff
@@ -0,0 +1,24 @@
+var wiersze = prompt('Podaj liczbę wierszy: '),
+	i,
+	k,
+	j;
+
+
+function rysujChoinke(n) {
+	for (i = 1, star = ''; i-1 != n; i++) {
```

Comment:
deklaracje `star = ''` dalbym poza forem
*proponuje zmienne do liczenia (i, k, j) zadeklarowac wewnatrz for (dodac po prostu var)